zerver/openapi/zulip.yaml

```diff
@@ -8420,6 +8420,29 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    delete:
+      operationId: delete-custom-emoji
+      summary: Delete custom emoji
+      tags: ["server_and_organizations"]
+      description: |
+        This endpoint is used to delete a custom emoji from the user's
+        organization.
+
+        Note: Deleted emoji will still be visible in old messages.
+
+        **Changes**: New in Zulip 7.0 (feature level 162).
```

Comment:
This line is wrong -- the API endpoint has existed since before feature level 1. Please delete it.

Reply:
Oh, I was thinking from the user's perspective as it will look like it is a newly added API endpoint 😅, nevermind removed it!